src/main/java/com/clussmanproductions/trafficcontrol/ModItems.java

```diff
@@ -27,6 +21,12 @@
 	public static ItemTrafficLight5Frame traffic_light_5_frame;
 	@ObjectHolder("traffic_light_doghouse_frame")
 	public static ItemTrafficLightDoghouseFrame traffic_light_doghouse_frame;
+	@ObjectHolder("traffic_light_one_frame")
+	public static ItemTrafficLightOneFrame traffic_light_one_frame;
```

Comment:
We should also use numerical "1" instead of textual "one"

Reply:
Fixed